Bottom: b03266668ec1aaf0e51f7ea46d430ee1c89867de
Top:    9cbf10f870975545bc2c57ad9caf62a852ac7d32
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-10 01:19:40 +0000

Refresh of add-split-colouring-for-find-a

---

diff --git a/src/action.c b/src/action.c
index 5e806b5..b5a65b3 100644
--- a/src/action.c
+++ b/src/action.c
@@ -2298,14 +2298,14 @@ ActionConnection (int argc, char **argv, Coord x, Coord y)
 	case F_ResetLinesAndPolygons:
 	  {
 	    bool change = false;
-	
+
 	    SaveFindFlag (RATFOUNDFLAG);
 	    change = ResetFoundLinesAndPolygons (true) || change;
 	    RestoreFindFlag ();
 	    SaveFindFlag (FOUNDFLAG);
 	    change = ResetFoundLinesAndPolygons (true) || change;
 	    RestoreFindFlag ();
-	
+
 	    if (change)
 	      {
 	        IncrementUndoSerialNumber ();
@@ -2324,7 +2324,7 @@ ActionConnection (int argc, char **argv, Coord x, Coord y)
 	    SaveFindFlag (FOUNDFLAG);
 	    change = ResetFoundPinsViasAndPads (true) || change;
 	    RestoreFindFlag ();
-	
+
 	    if (change)
 	      {
 	        IncrementUndoSerialNumber ();
@@ -2332,7 +2332,7 @@ ActionConnection (int argc, char **argv, Coord x, Coord y)
 	      }
 	    break;
 	  }
-	
+
 	case F_Reset:
 	  {
 	    bool change = false;
@@ -2343,7 +2343,7 @@ ActionConnection (int argc, char **argv, Coord x, Coord y)
 	    SaveFindFlag (FOUNDFLAG);
 	    change = ResetConnections (true) || change;
 	    RestoreFindFlag ();
-	
+
 	    if (change)
 	      {
 	        IncrementUndoSerialNumber ();
